recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - versioningit
   run:
     - click
-    - gufe ==0.7.*
+    - gufe ==0.7.2
```

Comment:
As per yesterday's call, we're pinning to 0.7.2 of gufe because (as far as we remember), we're not compatible with the 0.7.1 bugfix.

If we want to keep gufe & openfe in lockstep, we can do a gufe in the next few days and just make a new build here.

Reply:
This was probably 0.7.1<=x<0.8.0 but it's no drama.